arithmetic.py

```diff
@@ -1,26 +1,34 @@
-def addition(a,b):
+def addition(a, b):
+    """Returns the sum of the numbers a and b"""
     return a+b
 
```

Comment:
["Surround top-level function and class definitions with two blank lines"](https://www.python.org/dev/peps/pep-0008/#blank-lines). Just add one more line between each function definition :grin:.